app/forms/models/forms.py

```diff
@@ -173,8 +173,13 @@ class Meta:
 
     @classmethod
     def has_write_permission(cls, request):
-        group_slug = request.data.get("group")
-        group = Group.objects.filter(slug=group_slug).first()
+        if request.method == "POST":
+            group_slug = request.data.get("group")
+            group = Group.objects.filter(slug=group_slug).first()
+        else:
+            form_id = request.parser_context.get("kwargs", {}).get("pk", None)
+            form = GroupForm.objects.filter(id=form_id).first()
+            group = form.group if form else None
```

Comment:
Kan group være None? Bør ikke det endres i serializer at man ikke trenger å sende inn gruppen i en PUT reqeust.

Reply:
La til den sjekken kun for å håndtere når vi skal sjekke en bruker's permissions på alle modeller. Da vil ikke form_id være noe og vi vil ikke finne et form, da blir også group None, men det har ikke noe betydning. Group vil ikke være None i noen andre tilfeller